sidebars/sidebarsDevelopers.js

```diff
@@ -25,3 +25,12 @@ module.exports = {
     'v2/fork-yearn-ui',
   ],
 }
+{
+  type: 'category',
+  label: 'Subgraph Information',
+  items: [
+    'docs/developers/v2/subgraph info.md',
```

Comment:
Don't use spaces in filenames, use `-`

Reply:
Thank you @saltyfacu.  I am not the greatest w/ Sidebars.  I appreciate your feedback.  Should be fixed I believe.  Let me know.

Reply:
this is looking good. Is there any other place we can put yDaemon in? I think it has nothing to do with the subgraph